content/docs/cml/start-gitlab.md

```diff
@@ -0,0 +1,117 @@
+# Using CML on GitLab
+
+Here, we'll walk through a tutorial to start using CML on GitLab. For
+simplicity, we'll show the demo in GitLab Actions, but these instructions are
+valid for all supported Continuous Integration (CI) systems (with exceptions as
+noted!).
+
+1. Fork our
+   [example project repository](https://gitlab.com/iterative.ai/example_cml).
+   Click on Fork and select what namespace to fork the project to.
+
+   ![](/img/gitlab_fork_cml_project.png)
+
+   > 💡 If just starting to use GitLab, you may receive a warning at the top of
+   > the page regarding SSH keys. You do not need to use SSH keys for the
+   > tutorial so you can click: "Don't show me again."  
+   > [Click this link for more information on SSH keys.](https://docs.gitlab.com/ee/ssh/)
+
+2. ⚠️ CML in GitLab works exactly as in GitHub except for a few conventions. In
+   Gitlab, to use CML, you must create a variable called a repo_token whose
```

Comment:
Prob no need to compare with the Github process: people going into this page will likely skip that one.

Reply:
Also, 💅 the 2nd instance of "Gitlab" needs capital `L` (for consistency).